core/sail/spin/src/test/resources/functions/apf/concat.sparql

```diff
@@ -0,0 +1,5 @@
+prefix apf: <http://jena.hpl.hp.com/ARQ/property#>
```

Comment:
Conventionally file extension is .rq for sparql queries. I think it is mentioned in the sparql federation sparql if you are interested. Is it possible to have .srx files to check the results against? Check out rdf4j/testsuites/sparql/src/main/resources/testcases-sparql-1.1-w3c/functions to see how it can be done.